src/Command/Fixture/AddCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Command\Fixture;
+
+use App\Entity\Comment;
+use App\Entity\Post;
+use App\Entity\Tag;
+use App\Entity\User;
+use App\Repository\TagRepository;
+use Cycle\ORM\ORMInterface;
+use Cycle\ORM\Transaction;
+use Faker\Factory;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Yiisoft\Yii\Console\ExitCode;
+
+class AddCommand extends Command
+{
+    private ORMInterface $orm;
+
+    protected static $defaultName = 'fixture/add';
+
+    public function __construct(ORMInterface $orm)
+    {
+        parent::__construct();
+        $this->orm = $orm;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setDescription('Add fixtures')
+            ->setHelp('This command adds random content')
+            ->addArgument('count', InputArgument::OPTIONAL, 'Count', 10);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $count = $input->getArgument('count');
+        // get faker
+        if (!class_exists(Factory::class)) {
+            $io->error('Faker should be installed. Run `composer install --dev`');
+            return ExitCode::UNSPECIFIED_ERROR;
+        }
+        $faker = Factory::create();
+
+        // users
```

Comment:
That's a candidate for private method such as addUsers(). Same for parts w/ comments below.